packages/zksend/src/experimental/wallet/index.ts

```diff
@@ -48,19 +48,22 @@ const getStashedSession = (): { accounts: StashedAccount[]; token: string } => {
 };
 
 const SUI_WALLET_EXTENSION_ID = 'com.sui.extension' as const;
-
+const METADATA_API_URL = ' http://localhost:3001/api/wallet/metadata';
 export class StashedWallet implements Wallet {
 	#events: Emitter<WalletEventsMap>;
 	#accounts: ReadonlyWalletAccount[];
 	#origin: string;
+	#walletName: string = STASHED_WALLET_NAME; // Default name
```

Comment:
I _think_ for the initial release we can make dynamic config a requirement and follow-up with opt-outs + static fallbacks